### PR TITLE
Add an app name as id into registration apps

### PR DESCRIPTION
**Pull request by oleh-momot**

### CODE REVIEW

File: ilc/client.js [hunk 1]
```diff
@@ -19,57 +27,66 @@ const registryConf = JSON.parse(confScript.innerHTML);
...
-                     */
-                    const styleSheet = module.default;
-                    document.adoptedStyleSheets = [...document.adoptedStyleSheets, styleSheet];
-                }));
-            }
+registryConf.routes.map((route) => route.slots).forEach((slots) => {
+    Object.keys(slots).forEach((slotName) => {
+        const {
+            appName: slotAppName,
+            id: slotId,
```
Comment (StyleT) [inline ilc/client.js]: Use `slotName` instead of `slotId`

File: ilc/router/ServerRouter.js [hunk 2]
```diff
@@ -71,11 +71,14 @@ module.exports = class ServerRouter {
             }
 
             const url = new URL(ssrOpts.src);
+            const appName = `${slotData.appName.replace('@portal/', '')}${slotData.id}`;
 
             const reqProps = {
                 basePath: route.basePath,
-                reqUrl: route.reqUrl
+                reqUrl: route.reqUrl,
+                appName,
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply ilc/router/ServerRouter.js]: Updated

File: ilc/client.js [hunk 3]
```diff
@@ -19,57 +27,66 @@ const registryConf = JSON.parse(confScript.innerHTML);
...
-}
 
-function getMountPointFactory(appName) {
+        const appName = `${slotAppName.replace('@portal/', '')}${slotId}`;
+
+        singleSpa.registerApplication(
+            appName,
+            () => {
+                const waitTill = [System.import(slotAppName)];
+                const appConf = registryConf.apps[slotAppName];
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply ilc/client.js]: Updated

File: ilc/client.js (hunk 1 above)
Comment (oleh-momot) [reply ilc/client.js]: Updated

File: ilc/client.js [hunk 4]
```diff
@@ -19,57 +27,61 @@ const registryConf = JSON.parse(confScript.innerHTML);
...
-            getCurrentPathProps: getCurrentPathPropsFactory(appName),
-            getCurrentBasePath,
+        if (!registryConf.apps.hasOwnProperty(appName)) {
+            return;
         }
-    );
-}
 
-function getMountPointFactory(appName) {
+        const fragmentName = `${appName.replace('@portal/', '')}${slotName}`;
```
Comment (StyleT) [inline ilc/client.js]: `${appName.replace('@portal/', '')}@${slotName}`

File: ilc/router/ServerRouter.js [hunk 5]
```diff
@@ -71,11 +71,14 @@ module.exports = class ServerRouter {
             }
 
             const url = new URL(ssrOpts.src);
+            const fragmentName = `${slotData.appName.replace('@portal/', '')}${slotName}`;
```
Comment (StyleT) [inline ilc/router/ServerRouter.js]: same here

File: ilc/client.js (hunk 4 above)
Comment (StyleT) [reply ilc/client.js]: How it will look eventually: https://take.ms/3bbrb

File: ilc/router/ServerRouter.js (hunk 5 above)
Comment (oleh-momot) [reply ilc/router/ServerRouter.js]: Updated

File: ilc/client.js (hunk 4 above)
Comment (oleh-momot) [reply ilc/client.js]: Updated

**Pull request merged**